### PR TITLE
editorial: wallet instance attestation and solution

### PR DESCRIPTION
**Pull request by peppelinux**
## Review

- [ ] Ensure your files are written following RST specs (*not MD!*)
- [ ] Italian version
- [ ] English version
- [ ] Example files 
- [ ] Ask for review

### CODE REVIEW

**Review by balanza — APPROVED** [review]
LGTM

File: docs/en/wallet-instance-attestation.rst [hunk 1]
```diff
@@ -5,11 +5,14 @@
 Wallet Instance Attestation
 +++++++++++++++++++++++++++
 
-Inside a **Wallet Solution** and, especially with regards
-to the **Wallet Instance**, it is essential to ensure the **authenticity,
-integrity, security, privacy and trust** in the use of the latter both
-by the User and the services connected to it, such as the
-**PID Provider** or one **Relying Party**.
+The Wallet Instance Attestation contains data about the Wallet Provider, 
+the Wallet Solution, the Wallet  Instance and the security level of the device
```
Comment (ruphy) [inline docs/en/wallet-instance-attestation.rst]:
```suggestion
the Wallet Solution, the Wallet Instance and the security level of the device 
```

**Review by ruphy — APPROVED** [review]

File: docs/en/relying-party-solution.rst [hunk 2]
```diff
@@ -497,7 +497,7 @@ Below is a non-normative response example:
             ]
         },
         "metadata": {
-            "wallet_verifier": {
+            "wallet_relying_party": {
```
Comment (peppelinux) [inline docs/en/relying-party-solution.rst:500]: @grausof @balanza changes according to yesterday editor call, to get aligned to the one already changedi n the trust model section

**Review by grausof — APPROVED** [review]
LGTM

**Review by fmarino-ipzs — APPROVED** [review]

**Pull request merged**